bluechi.spec.in

```diff
@@ -244,6 +295,7 @@ will be used during integration tests when creating code coverage report.
 %endif
 
 
+
```

Comment:
Do we need this 3rd empty line?

Reply:
No, I added it just to have a bit of "separation" between defining the packages and the actual build steps.